test/libsolidity/ASTJSON/assembly/call.json

```diff
@@ -0,0 +1,168 @@
+{
+  "absolutePath": "a",
+  "exportedSymbols":
+  {
+    "C":
+    [
+      6
+    ]
+  },
+  "id": 7,
+  "nodeType": "SourceUnit",
+  "nodes":
+  [
+    {
+      "abstract": false,
+      "baseContracts": [],
+      "contractDependencies": [],
+      "contractKind": "contract",
+      "documentation": null,
+      "fullyImplemented": true,
+      "id": 6,
+      "linearizedBaseContracts":
+      [
+        6
+      ],
+      "name": "C",
+      "nodeType": "ContractDefinition",
+      "nodes":
+      [
+        {
+          "body":
+          {
+            "id": 4,
+            "nodeType": "Block",
+            "src": "37:59:1",
+            "statements":
+            [
+              {
+                "AST":
```

Comment:
Actually what's this `AST` key?

Reply:
It's a node-type-specific key for the node type "InlineAssembly".

Reply:
Actually - it would probably be better to prefix all the Yul AST node types by `Yul` wouldn't it?

Reply:
It sounds quite generic. Why not call it something closer to `InlineAssembly`?

Reply:
> Actually - it would probably be better to prefix all the Yul AST node types by `Yul` wouldn't it?

Yea I agree

Reply:
But the keys that depend on the node type are always "generic" because they are only seen in the context of the node type.

Reply:
Hmm ok